packages/mobile/src/screens/chat-screen/ChatBlastCTA.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback } from 'react'
+
+import { useGetCurrentUserId } from '@audius/common/api'
+import { useSelectTierInfo } from '@audius/common/hooks'
+import { useChatBlastModal } from '@audius/common/store'
+import { TouchableHighlight } from 'react-native-gesture-handler'
+
+import {
+  Box,
+  Flex,
+  Text,
+  IconTowerBroadcast,
+  IconCaretRight,
+  useTheme,
+  IconVerified,
+  IconTokenBronze
+} from '@audius/harmony-native'
+
+const messages = {
+  title: 'Send a Message Blast',
+  description: 'Send messages to your fans in bulk.',
+  badgeRequired: 'Badge Required',
+  or: 'or'
+}
+
+type ChatBlastCTAProps = {
+  onClick: () => void
+}
+
+export const ChatBlastCTA = (props: ChatBlastCTAProps) => {
+  const { onClick } = props
+
+  const { onOpen: openChatBlastModal } = useChatBlastModal()
+
+  const { data: userId } = useGetCurrentUserId({})
+  const { tierNumber, isVerified } = useSelectTierInfo(userId ?? 0) ?? {}
```

Comment:
wondering if it makes sense to make this hook accept optionaly `userId` and just return early if it's undefined. easier than always having to `?? 0`. but doesn't need to be this PR.